portlets/src/main/webapp/skin/less/challenges.less

```diff
@@ -82,6 +82,14 @@
 
 #engagementCenterProgramDetail {
   min-height: ~"calc(100vh - 170px)";
+  #rulesListToolbar {
+    .v-toolbar__content {
+      padding: 0!important;
+    }
+    .v-input__icon--clear .v-icon {
+      font-size: 16px !important;
+    }
```

Comment:
can you please delete this specific styles. It can be treated using different ways like using `<v-toolbar dense` `<v-icon size="16"` `class="mx-n2"`...